app/views/pages/statistics.ejs

```diff
@@ -8,6 +15,9 @@
           <canvas id="symptomsChart"></canvas>
           <h2>Antall rapporteringer</h2>
           <canvas id="totalReportsChart"></canvas>
+          <h2>Nærkontakt med smittede</h2>
+          <p>Totalt har <%- peopleInContactWithInfected -%> personer rapportert at de har hatt nærkontakt med en person som har testet positivt på COVID-19.</p>
```

Comment:
Når du bruker `<%-` i steden for `<%=`, renderer den også dangerous characters (potensielt `<script>` osv). Sikkert greit å holde seg til `<%=`de stedene vi ikke er nødt til å bruke `<%-` (som når vi skriver ut JSON).